src/components/pages/rules/flyouts/index.js

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 export * from './ruleDetails';
-export * from './ruleNew';
+export * from './ruleNew/ruleNew';
```

Comment:
This naming pattern confuses me. Why ./ruleNew/ruleNew? May be just keep it ./ruleNew

Reply:
The reason for this is component can both a .js file and a .scss file. Placing them in a folder helps keep them grouped. It is also a pattern throughout the rest of the UI. You can create index.js files to help deep with the import/export issue inside the component folder but, since there is only one component in there, it doesn't seem worth it. The flyout component will imported from this file anyways so this is the only place this double naming export will appear.